src/controllers/roomController.js

```diff
@@ -0,0 +1,110 @@
+import models from '../models';
+import { successResponse, errorResponse } from '../utils';
+
+const { Rooms, Accommodations } = models;
+/**
+ * @class RoomsController
+ * @description Controllers for handling rooms requests
+ * @exports AccommodationController
+ */
+class RoomController {
+  /**
+    * @method createRooms
+    * @description Method to create room facilities
+    * @param {object} req - The Request Object
+    * @param {object} res - The Response Object
+    * @returns {object} Newly created room details
+    */
+  static async createRoom(req, res) {
+    const {
+      type, capacity, image, accommodationId
```

Comment:
You should not be sending `accommodationId` in the request body. The route to creating a room for a particular accommodation should look like this:
`POST /api/v1/accommodations/accommodationId/rooms`
to get all the rooms for a particular accommodation:
`GET /api/v1/accommodations/accommodationId/rooms`
for updating a room:
`PUT /api/v1/rooms/roomId`
to get a particular room:
`GET /api/v1/rooms/roomId`
to update a room
`PUT /api/v1/rooms/roomId`
to delete a room
`DELETE /api/v1/rooms/roomId`